webpack/scenes/ContentViews/Details/Promote/ContentViewVersionPromote.js

```diff
@@ -94,6 +101,10 @@ const ContentViewVersionPromote = ({
 
   const submitDisabled = loading || userCheckedItems.length === 0;
 
+  if (redirect && detailsPage) {
+    return (<Redirect to="/versions" />);
```

Comment:
why not just use history.push here? instead of rendering a redirect component with state?

Reply:
I tried history.push and react wasn't too happy..It worked but with a bunch of errors in the console.